platform.md

```diff
@@ -148,6 +159,441 @@ A platform MAY support any number of mixins for a given stack in order to suppor
+- IF `<run-image>` is not provided by the platform the value will be [resolved](#run-image-resolution) from the contents of the `stack` key in the `io.buildpacks.lifecycle.metdata` label on `<image>`.
+
+| Output             | Description
+|--------------------|----------------------------------------------
+| [exit status]      | Pass (0), or error (1+)
+| `/dev/stdout`      | Logs (info)
+| `/dev/stderr`      | Logs (warnings, errors)
+| `<image>`          | Rebased app image (see [Buildpack Interface Specfication](buildpack.md)
+
+- The lifecycle SHALL write the same app image to each `<image>` tag
+- The rebased app image SHALL be identical to `<image>`, with the following modifications:
+    - Run image layers SHALL be defined as Layers in `<image>` up to and including the layer with diff ID matching the value of `run-image.top-layer` from the `io.buildpacks.lifecycle.metadata` label
+    - Run image layers SHALL be replaced with the layers from the new `<run-image>`
+    - The value of `io.buildpacks.lifecycle.metadata` SHALL be modified as follows
+      - `run-image.reference` SHALL uniquely identify `<run-image>`
+      - `run-image.top-layer` SHALL be set to the uncompressed digest of the top layer in `<run-image>`
+
+#### `launcher`
+Usage:
+```
+/cnb/lifecycle/launcher [--] [<cmd> <arg>...]
+```
+
+| Input               | Environment Variable  | Default Value         | Description
+|---------------------|-----------------------|-----------------------|---------------------------------------
+| `<app>`             | `CNB_APP_DIR`         | `/workspace`          | Path to application directory
+| `<layers>`          | `CNB_LAYERS_DIR`      | `/layers`             | Path to layer directory
+| `<process-type>`    | `CNB_PROCESS_TYPE`    | `web`                 | `type` of process to launch
+| `<direct>`          |                       | `false`               | Execution strategy for user provided process
+| `<cmd>`             |                       |                       | User-provided command
+| `<arg>...`          |                       |                       | Arguments to user-provided command
+| `<layers>/config/metadata.toml`    |        |                       | Build metadata (see [`metadata.toml`](#metdata.toml-(toml))
+| `<layers>/<buildpack-id>/<layer>/` |        |                       | Launch Layers
+
+- IF `$1` is `--` `<direct>` is `true` and `<args>` SHALL be `${@2:}`
+- IF `$1` is anything other than `--`, `<direct>` is `false`, and `<args>` SHALL be `$@`
+- IF `<direct>`
+- IF `<args>` are NOT provided `launcher` SHALL select the process with `type` equal to `<process-type>` from `<layers>/config/metadata.toml` and launch it as described in the [Buildpack Interface Specification](buildpack.md).
+- IF `<args>` are provided `launcher` SHALL behaves as if it selected a process with the following process from `metadata.toml`.
+```
+[[process]]
```

Comment:
Should this be:
```suggestion
[process]
```

Reply:
Given that `metadata.toml` contains a list of processes I think giving this example as a list item is correct. I think the mistake here is that it should actually be `[[processes]]`.